docs/modules/ROOT/pages/how-to/zone-setup.adoc

```diff
@@ -0,0 +1,66 @@
+= Activate {zone}
+
+This guide describes the steps required to turn an OpenShift 4 cluster into an {zone}.
+
+== Configure group-sync-operator
+
+The https://github.com/appuio/keycloak-attribute-sync-controller[group-sync-operator] is required to sync Keycloak groups to each {zone}.
```

Comment:
Didn't we discuss having a single user that's used to sync all zones? Or is that something we don't want to document here but somewhere else, specifically for VSHN's APPUiO Cloud setup?

Reply:
Oops, the comment was supposed to be for the next line of text which I read as "create a sync user for each zone" since it's part of the zone activation documentation.

Reply:
ah! Well that I did mess up :)